mp_cnn/dataset.py

```diff
@@ -6,6 +6,7 @@
 from datasets.sick import SICK
 from datasets.msrvid import MSRVID
 from datasets.trecqa import TRECQA
```

Comment:
The casing of TrecQA and WikiQA are different here. Not a big deal, you can ignore it.

Reply:
Yeah, I did it deliberately since TREC is an abbreviation: Text REtrieval Conference (TREC) while Wiki is not. Not sure what's the best thing to do here.